### PR TITLE
Update to job submission script

### PR DESCRIPTION
**Pull request by jbrillon**
Updating job submission script with memory per node parameter for faster queue times

### CODE REVIEW

File: generate_job_parameters_file_PHiLiP.sh [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # This file quickly generates the job paramters file
 
 filename="my_job_parameters_file.sh"
-time="2:00:00" # wall time
+walltime="2:00:00" # wall time
 job_name="run_parameters_file" # slurm job name
 nodes=1 # number of nodes
-ntasks_per_node=40 # number of processors per node; refer to https://docs.computecanada.ca/wiki/Advanced_MPI_scheduling
+ntasks_per_node=16 # number of processors per node; refer to https://docs.computecanada.ca/wiki/Advanced_MPI_scheduling
+memory_per_node="63G" # requested memory per node; NOTE: Must correspond to ntasks_per_node; refer to https://docs.computecanada.ca/wiki/Advanced_MPI_scheduling
```
Comment (cpethrick) [inline generate_job_parameters_file_PHiLiP.sh:8]: Maybe rephrase comment to `NOTE: Must correspond to n_tasks_per_node * memory_per_core` to reduce confusion for new users

**Review by cpethrick — APPROVED** [review]
Thanks for updating this, it is very helpful!

**Review by PranshulThakur — APPROVED** [review]
LGTM!

File: generate_job_parameters_file_PHiLiP.sh (hunk 1 above)
Comment (jbrillon) [inline generate_job_parameters_file_PHiLiP.sh:8]:
```suggestion
memory_per_node="63G" # requested memory per node; NOTE: Must correspond to ntasks_per_node*memory_per_core; refer to https://docs.computecanada.ca/wiki/Advanced_MPI_scheduling
```

Comment (jbrillon) [reply generate_job_parameters_file_PHiLiP.sh]: good call, its been fixed
